scripts/release.js

```diff
@@ -111,14 +110,41 @@ fs.writeFileSync(
 spawnOrFail('npm', [`version ${versionString} --no-git-tag-version`]);
 logger.log(`Updated package.json version to ${versionString}`);
 
+// Update the peer dependency to the most updated version of the SDK
+const updatedSdkVersion = spawnOrFail('npm', [`show amazon-chime-sdk-js version`]).trim();
+
+logger.log(`Installing SDK Version: ${updatedSdkVersion} into the meeting demo, chat demo, and as a peerDependency and devDependency of the react library.`);
+
+let componentsPackageJson = JSON.parse(fs.readFileSync('package.json', 'utf-8'));
+const oldSdkVersion = componentsPackageJson.peerDependencies['amazon-chime-sdk-js'];
+let replacedPeerDependency = fs.readFileSync('package.json').toString();
+// Replace first instance
+replacedPeerDependency = replacedPeerDependency.replace(`"amazon-chime-sdk-js": "${oldSdkVersion}"` , `"amazon-chime-sdk-js": "^${updatedSdkVersion}"`);
+// Replace second instance
+replacedPeerDependency = replacedPeerDependency.replace(`"amazon-chime-sdk-js": "${oldSdkVersion}"` , `"amazon-chime-sdk-js": "^${updatedSdkVersion}"`);
+fs.writeFileSync(
+  'package.json',
+  replacedPeerDependency
+);
+
+logger.log("NPM Installing component library...")
+spawnOrFail('npm', [`install`]);
+
+// Udpate meeting demo to the most up to date version of the SDK
+process.chdir(path.join(__dirname, '../demo/meeting'));
+spawnOrFail('npm', [`install amazon-chime-sdk-js@${updatedSdkVersion}`]);
```

Comment:
What is the outcome of this command? Does it update the dependency to ^2.5.0 from ^2.4.1?
Also I think we want to run npm run build here, otherwise the deployment script may pick up the old dist.

Reply:
The deploy script always run npm run build before deploying, so I dont think we need to build here. The updatedSdkVersion should be the newest version that currently exists in NPM.

Reply:
Ok did not know that deploy script in component is different from js sdk. 
> What is the outcome of this command? Does it update the dependency to ^2.5.0 from ^2.4.1?

What about this?

Reply:
> Does it update the dependency to ^2.5.0 from ^2.4.1?

Yes, it will overwrite it.